### PR TITLE
[FLINK-3009] Add dockerized jekyll environment

### PR DESCRIPTION
**Pull request by jaoki**
This is not to fulfill what FLINK-3009 is originally asking,
but I'm piggybacking to contribute a docker environment with jekyll
so that users won't have to deal with environment versions.

### CODE REVIEW

**Comment by StephanEwen** [comment]
Useful addition, +1 from my side

File: docs/docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+RUN groupadd --non-unique -g ${GROUP_ID} ${USER_NAME} && \
+  useradd -g ${GROUP_ID} -u ${USER_ID} -k /root -m ${USER_NAME}
+ENV  HOME /home/${USER_NAME}
+UserSpecificDocker
+
+FLINK_DOC_ROOT=${SCRIPT_DIR}/..
+
+CMD="
+echo
+echo 'Welcome to Flink docs'
```
Comment (hsaputra) [inline docs/docker/run.sh]: Small nit: Apache Flink

**Comment by hsaputra** [comment]
Other than small nit on text, +1

Seems like Travis fails unrelated to this patch

File: docs/docker/run.sh (hunk 1 above)
Comment (jaoki) [inline docs/docker/run.sh]: @hsaputra , thank you for your feedback. Will fix.

**Comment by jaoki** [comment]
Does anyone want to commit this?

**Comment by hsaputra** [comment]
LGTM 
+1
Will commit end of day if no one beats me

**Pull request closed without merging**